vmdb/app/helpers/application_helper.rb

```diff
@@ -950,7 +950,13 @@ def build_toolbar_hide_button(id)
         return true if requester.name != @record.requester_name || ["approved", "denied"].include?(@record.approval_state)
       when "miq_request_copy"
         requester = User.find_by_userid(session[:userid])
-        return true if !["MiqProvisionRequest", "MiqHostProvisionRequest"].include?(@record.resource_type) || ((requester.name != @record.requester_name || ["approved", "denied"].include?(@record.approval_state)) && @showtype == "miq_provisions")
+        resource_types_for_miq_request_copy = %w(MiqProvisionRequest
+                                                 MiqHostProvisionRequest
+                                                 MiqProvisionConfiguredSystemRequest)
+        return true if !resource_types_for_miq_request_copy.include?(@record.resource_type) ||
+                       ((requester.name != @record.requester_name ||
+                        %w(approved denied).include?(@record.approval_state)) &&
```

Comment:
@tenderlove I found this UI Constant 
```
  PROV_STATES = {
                  "pending_approval"  => "Pending Approval",
                  "approved"          => "Approved",
                  "denied"            => "Denied"
                }
```
I suppose we could use that instead of using the strings "approved" and "denied"
```
@record.approval_state == PROV_STATES.keys[1] || PROV_STATES.keys[2]
```

Would that be a good alternative ?